fivetran_provider_async/operators.py

```diff
@@ -21,8 +22,10 @@ class RegistryLink(BaseOperatorLink):
     def get_link(self, operator, dttm):
         """Get link to registry page."""
 
-        registry_link = "https://registry.astronomer.io/providers/{provider}/modules/{operator}"
-        return registry_link.format(provider="fivetran", operator="fivetranoperator")
+        return (
+            f"https://registry.astronomer.io/providers/airflow-provider-fivetran-async/versions/"
+            f"{__version__}/modules/FivetranOperator"
```

Comment:
wondering why we are not using operator variable and hardcoding FivetranOperator?

Reply:
If someone is still using deprecated FivetranOperatorAsync would it cause problem?

Reply:
I think previously it was hardcoded too to FivetranOperator.

Reply:
hmm, but look like the operator is an object here so `operator.__class__.__name__`  should work

Reply:
+ 1. This would make it extensible to the sensor as well.